### DISCUSSION TITLE
How to access translated page in tldr client(linux) in terminal ?

### DISCUSSION THREAD

**Discussion by kbdharun** [tldr-pages/tldr#8270] [Q&A] [answered]
Is there any commands for tldr (linux) that could fetch and display a translated version of page in terminal. Is there any implementation for it. Or does it display a tranlated version of page based on system language?

**Comment by navarroaxel** [comment, marked as answer]
If you check the client specification https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#arguments, the clients could implement the optional `--language` argument.
If you want the Spanish version of a page you can use:
```bash
tldr --language es git
```
But the tldr client should fallback to English If that translation doesn't exist.

Also the clients use the `LANG` env var to know if they should use a more specific language instead of English https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#language

**Reply by kbdharun** [reply]
Can this argument be added to https://github.com/tldr-pages/tldr/blob/main/pages/common/tldr.md ? 
Even though it can be accessed through `tldr --help` command , adding it to tldr.md would be helpful too

**Reply by navarroaxel** [reply]
In the `tldr --help` every client can add custom arguments only available in that implementation.
About the `tldr tldr` page, only the required arguments should be documented there.

**Comment by sbrl** [comment]
In addition to @navarroaxel's answer, you can set the `LANG` and `LANGUAGE` environment variables: https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#language

This has the advantage of setting your language preferences once for all programs, and then it will be followed where possible (given translation is a work-in-progress - open a PR to help out!).

These are well-known environment variables supported by a large number of different programs.
